Scheduler - Getting Started: Add ASP.NET Core Snippets (#9038)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/00 Getting Started with Scheduler.md

The Scheduler component emulates the user interface of Calendar applications on Windows or Mac OS.

> This tutorial adds DevExtreme Scheduler to a page, binds the component to data, and configures core Scheduler features.

Reviewer: ```suggestion
This tutorial shows how to add DevExtreme Scheduler to a page, bind it to data, and configure its core features.
```

<div class="simulator-desktop-container" data-view="/Content/Applications/26_2/GettingStartedWith/Scheduler/index.html, /Content/Applications/26_2/GettingStartedWith/Scheduler/index.js, /Content/Applications/26_2/GettingStartedWith/Scheduler/index.css" style="border: none !important;"></div>

> Each section in this tutorial covers a single configuration step. You can also find the full code in the following GitHub repository:

Reviewer: ```suggestion
Each section covers a single configuration step. The complete code is available in the following GitHub repository:
```
